kotlin/build.gradle

```
@@ -9,7 +9,7 @@ buildscript {
     }
     dependencies {
         classpath "com.android.tools.build:gradle:$gradleVersion"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion" }
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"}
```

uh oh, this ol chestnut again :p
```suggestion
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
        }
```